enterprise/core-edge/src/main/java/org/neo4j/coreedge/discovery/HazelcastCoreTopologyService.java

```diff
@@ -206,15 +212,17 @@ public void memberAdded( MembershipEvent membershipEvent )
         {
             log.info( "Core member added %s", membershipEvent );
             log.info( "Current core topology is %s", coreServers() );
-            listenerService.notifyListeners( coreServers());
+            listenerService.notifyListeners( coreServers() );
```

Comment:
Need to call refreshCoreTopology() before logging, otherwise the log message will have the old topology, which will be very confusing. Maybe move the logging inside refreshCoreTopology() ?